01-Plot-Raster-In-R.Rmd

```diff
@@ -0,0 +1,346 @@
+---
+layout: post
```

Comment:
Because this file has a new name, i can't see the individual changes  to the file (unless i go into each commit). Please change it back? Then we can submit an issue to change the names of ALL FILES in the repo.